### ISSUE TITLE
Closing bracket from classes not "linting"

### ISSUE THREAD

**Issue by crazydisi** [rodjek/puppet-lint#250]
Hi,

im quite not sure if its an error but when i forget the closing bracket within a class definition, puppet-lint does not show an error.

Here a invented example

```
class ntp ($param1 = '') {

  package { 'ntp':
    ensure => installed,
  }

  file { '/etc/ntp.conf':
    ensure  => file,
    content => template('ntp/ntp.conf.erb'),
    owner   => 'root',
    group   => 'root',
    mode    => '0644',
    require => Package['ntp'],
  }


```
